### PR TITLE
Moving clang-format back to main tests

### PR DESCRIPTION
**Pull request by parsoyaarihant**
* Since `LLVM apt repository` is back up, the allowed failures are removed.

### CODE REVIEW

**Comment by Sumith1896** [comment]
Looks good to me

**Comment by isuruf** [comment]
Looks good. Can you merge with master and then run the formatting script?

File: symengine/flint_wrapper.h [hunk 1]
```diff
@@ -1,731 +1,558 @@
 #ifndef SYMENGINE_FLINT_WRAPPER_H
 #define SYMENGINE_FLINT_WRAPPER_H
 
-#include <symengine/symengine_rcp.h>
 #include <gmp.h>
+#include <symengine/symengine_rcp.h>
```
Comment (isuruf) [inline symengine/flint_wrapper.h]: Are you using clang-format version 3.7?

Comment (parsoyaarihant) [inline symengine/flint_wrapper.h]: I am using 3.8.

Comment (isuruf) [inline symengine/flint_wrapper.h]: Use 3.7 which is the version all of us are using.

**Comment by parsoyaarihant** [comment]
I have opened another PR #1004 since this PR has many unnecesary commits.

**Pull request closed without merging**